src/translations/fr.json

```diff
@@ -861,7 +861,7 @@
     "Substation": "Site",
     "BusBarSections": "Sections de jeu de barre",
     "BusBarCount": "Nb de barres",
-    "SectionCount": "Nb de sections",
+    "voltageSectionCount": "Nb de sections",
```

Comment:
I think that we should find an other name for this key

Reply:
This is the key used in backend...